src/Microsoft.Azure.ServiceBus/SessionHandlerOptions.cs

```diff
@@ -83,11 +83,14 @@ public int MaxConcurrentSessions
                 this.MaxConcurrentAcceptSessionCalls = Math.Min(value, 2 * Environment.ProcessorCount);
             }
         }
-
+        
         /// <summary>Gets or sets whether the autocomplete option of the session handler is enabled.</summary>
         /// <value>true if the autocomplete option of the session handler is enabled; otherwise, false.</value>
         public bool AutoComplete { get; set; }
 
+        /// <summary>Gets or sets the optional session ID</summary>
```

Comment:
Assuming this PR goes through, I'd add a remark related to my comment about pinning to a single session.

Reply:
I didn't know exactly what you meant. But I've extended the summary in the latest commit.

Reply:
Changed it.